sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveSparkSubmitSuite.scala

```diff
@@ -108,8 +108,8 @@ class HiveSparkSubmitSuite
     val unusedJar = TestUtils.createJarWithClasses(Seq.empty)
     val jar1 = TestUtils.createJarWithClasses(Seq("SparkSubmitClassA"))
     val jar2 = TestUtils.createJarWithClasses(Seq("SparkSubmitClassB"))
-    val jar3 = TestHive.getHiveContribJar().getCanonicalPath
-    val jar4 = TestHive.getHiveHcatalogCoreJar().getCanonicalPath
+    val jar3 = HiveTestUtils.getHiveContribJar.getCanonicalPath
+    val jar4 = HiveTestUtils.getHiveHcatalogCoreJar.getCanonicalPath
     val jarsString = Seq(jar1, jar2, jar3, jar4).map(j => j.toString).mkString(",")
```

Comment:
It should throw an `AnalysisException`  if we remove jar3 and jar4. But it is successful now:
```
[info]   2019-09-04 22:54:30.372 - stderr> 19/09/04 22:54:30 WARN Master: App app-20190904225430-0000 requires more resource than any of Workers could have.
[info]   2019-09-04 22:54:37.434 - stderr> Exception in thread "main" org.apache.spark.sql.AnalysisException: Can not load class 'org.apache.hadoop.hive.contrib.udaf.example.UDAFExampleMax' when registering the function 'example_max', please make sure it is on the classpath;
[info]   2019-09-04 22:54:37.435 - stderr>
```